docs/paper/admin/how-to/anti-xray.md

```diff
@@ -280,3 +280,45 @@ anticheat:
 ```
 
 </details>
+
+## FAQ, Common Pitfalls and Support
+
+<details>
+  <summary>I can still see (some) ores / use X-ray</summary>
+
+As described above, there are several reasons why you might still see (some) ores even though you
+have enabled Anti-Xray:
+
+* The ores are above the configured `max-block-height` value.
+* Anti-Xray cannot hide ores exposed to air or other transparent blocks (in caves for example). In
+  principle this is also the case for `engine-mode: 2`, however, usually the fake ores obstruct the
+  view of real blocks. Hiding those exposed ores too requires additional plugins.
+* The `use-permission` option is enabled and you have the Anti-Xray bypass permission or you have
+  operator status.
+* The block type is missing in the configured block lists. This can be the result of using an
+  outdated configuration file.
+
+</details>
+
+<details>
+  <summary>It still doesn't work, further troubleshooting</summary>
+
+* Make sure to always restart your server after making changes to the Anti-Xray configuration.
+  Changes won't be applied automatically.
+* Do not use the `/reload` command. To apply Anti-Xray configuration changes a restart is required.
+* After restarting the server, verify that the configuration is applied correctly by inspecting the
+  config sections with Aikar's Minecraft Timings Viewer or spark.
```

Comment:
do we call it `Aikar's Minecraft Timings Viewer` elsewhere? that seems like a mouthfully, I would say with timings or spark, or with the timings command or via the timings viewer.